js/private/js_binary.bzl

```diff
@@ -575,13 +575,10 @@ def _js_binary_impl(ctx):
     providers = []
 
     if ctx.attr.testonly and ctx.configuration.coverage_enabled:
-        # We have to instruct rule implementers to have this attribute present.
-        if not hasattr(ctx.attr, "_lcov_merger"):
-            fail("_lcov_merger attribute is missing and coverage was requested")
```

Comment:
This issue will still exist in bazel <8, yet now we have no `fail()` informing users? I assume that's worth it but 🤔